app/drawing/profilecard/master.py

```diff
@@ -0,0 +1,165 @@
+import sys
+import os
+from io import BytesIO
+
+from PIL import Image, ImageDraw, ImageOps, ImageFont, ImageEnhance, ImageFilter
+import requests
+
+from app.utilities import text
+
+from ..utils import *
```

Comment:
Don't use relative paths.
```suggestion
from app.drawing.utils import *
```